pages/dao/[symbol]/proposal/components/instructions/SetMangoDepositoriesRedeemableSoftCap.tsx

```diff
@@ -137,6 +140,18 @@ const SetMangoDepositoriesRedeemableSoftCap = ({
         }
         error={formErrors['global']}
       />
+      <Input
+        label="Controller Address"
```

Comment:
Actually with the current program there can only be ONE controller
as such, controler + depositories etc can always be derived from that central controller

Reply:
Maybe derive the controller from this, and in the register controller section let the user select from a list of mint (sol, btc, msol w/e). Then you can derive Depository from Mint and ProgramID
Anywhere where a depository is asked it need to ask for a mint from a dropdown list optimally (just support SOL, BTC, MSOL for now, we add later)

Same for insurance a list with only USDC

Reply:
Indeed that sounds much better. I'll make this happen

Reply:
yeah does not make sense to have it as an input (and it is annoying to retrieve from previous instructions).
Will remove and get the derive address directly